docs/frontend-system/building-plugins/04-built-in-data-refs.md

```diff
@@ -0,0 +1,103 @@
+---
+id: built-in-data-refs
+title: Built-in data refs
+sidebar_label: Built-in data refs
+# prettier-ignore
+description: Configuring or overriding built-in extension data references
+---
+
+> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
+
+To have a better understanding of extension data references please read [the corresponding architecture section](../architecture/03-extensions.md#extension-data) first.
+
+## Built-in extension data references
+
+Data references help to define the inputs and outputs of an extension. A data ref is uniquely identified through its `id`. Through the data ref, strong typing is enforced for the input/output of the extension.
+
+### Core Extension Data
```

Comment:
Hmm, tbh I think it's enough to only document these three. The other data refs are owned by their respective extension types and are best documented in that section (if at all - they're encapsulated in the extension creator).

With that I think it could be good to break up the table into separate a separate section for each data ref, listing the type and ID as now, along with a brief description of the purpose and when to use it.